src/query/datavalues/tests/it/data_schema.rs

```diff
@@ -0,0 +1,59 @@
+// Copyright 2023 Datafuse Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use common_datavalues::prelude::*;
+use common_exception::Result;
+use pretty_assertions::assert_eq;
+
+#[test]
+fn test_project_schema_from_struct() -> Result<()> {
+    let child_field11 = u64::to_data_type();
```

Comment:
hmm, there is no to_data_type trait anymore.

Reply:
ok, use `DataTypeImpl::UInt64(UInt64Type)` instead.